test/test_vtk.py

```diff
@@ -27,3 +27,17 @@ def test_vtktositk():
     gc.collect()
 
     assert sitkimg[0, 0, 0] == 255.0
+
+def test_multichannel():
+    img = sitk.Image([10, 10], sitk.sitkVectorUInt8, 3)
+    img[0,0] = (255, 127, 42)
+    vtk_img = sitk2vtk(img)
+    print(vtk_img)
```

Comment:
Please remove the print, doesn't help with understanding what went wrong when something isn't as expected.